src/js/modules/ListLink.js

```diff
@@ -6,13 +6,12 @@ class ListLink extends Component {
 
   render() {
 
-    const { id, url, text, isBoxType } = this.props;
+    const { url, text, linkStyle } = this.props;
 
     return (
       <Link
-        className={ `coa-ListLink ${(isBoxType && "coa-ListLink--boxprimary" )}` }
-        key={id}
-        to={getPathWithLangCode(url)}
```

Comment:
Removed getPathWithLang because the url data has already been parsed when data is fetched.